generator/AWSPSGeneratorLib/HelpMaterials/Examples/LM/Remove-LMPermission.xml

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!--Examples for the AWS PowerShell CmdLets-->
+<examples>
+  <example>
+    <code>$json =  Get-LMPolicy -FunctionName "MylambdaFunction123" -Select Policy | ConvertFrom-Json| Select-Object -ExpandProperty Statement
+C:\> Remove-LMPermission -FunctionName "MylambdaFunction123" -StatementId $json[0].Sid</code>
```

Comment:
`$policy` would be a better name for the variable.
Please add `PS C:\>` in front of both lines of code.